osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -43,7 +43,13 @@ public partial class BeatmapListingOverlay : OnlineOverlay<BeatmapListingHeader>
 
         private Container panelTarget;
         private FillFlowContainer<BeatmapCard> foundContent;
-        private BeatmapListingFilterControl filterControl;
+
+        private BeatmapListingFilterControl filterControl => Header.FilterControl.With(f =>
```

Comment:
This is not good. A property which returns a child's property but also applies changes to it?

Find a way to do the event assignments inline in `load` or `LoadComplete` and probably just remove this.